app/controllers/api/projects.php

```diff
@@ -601,7 +601,7 @@
             throw new Exception('Project not found', 404, Exception::PROJECT_NOT_FOUND);
         }
 
-        $security = ($security === '1' || $security === 'true' || $security === 1 || $security === true);
+        $security = (bool) filter_var($security, FILTER_VALIDATE_BOOLEAN);
```

Comment:
This is not the only place we need to update. Search for `$search =` on all files.